Library/.rubocop.yml

```
@@ -15,7 +15,7 @@ AllCops:
   TargetRubyVersion: 3.1
   NewCops: enable
   Include:
-    - "**/*.rbi"
+    - "Homebrew/**/*.rbi"
   Exclude:
     - "Homebrew/sorbet/rbi/gems/**/*.rbi"
```

```suggestion
    - "Homebrew/sorbet/rbi/gems/**/*.rbi"
    - "Homebrew/sorbet/rbi/dsl/**/*.rbi"
```

This might be useful too since `rbi/dsl` are all generated.